src/AllocateParent.js

```diff
@@ -175,13 +143,62 @@ class AllocateParent extends Component {
 
 AllocateParent.propTypes = {
   error: PropTypes.string,
-  unallocatedOffenders: PropTypes.array,
-  autoAllocatedOffenders: PropTypes.array,
-  savedOffenders: PropTypes.array,
+  page: PropTypes.number,
+  unallocatedList: PropTypes.array,
+  allocatedList: PropTypes.array,
   allocatedKeyworkers: PropTypes.array,
   jwt: PropTypes.string.isRequired,
   onFinishAllocation: PropTypes.func.isRequired,
-  agencyId: PropTypes.string.isRequired
+  agencyId: PropTypes.string.isRequired,
+  fromDate: PropTypes.string,
+  toDate: PropTypes.string,
+  unallocatedListDispatch: PropTypes.func.isRequired,
+  allocatedDetailsDispatch: PropTypes.func.isRequired,
+  manualOverrideDispatch: PropTypes.func.isRequired,
+  manualOverrideDateFilterDispatch: PropTypes.func.isRequired,
+  setCurrentPageDispatch: PropTypes.func.isRequired,
+  setErrorDispatch: PropTypes.func.isRequired
+};
+
+const mapStateToProps = state => {
+  return {
+    unallocatedList: state.unallocated.unallocatedList,
+    page: state.app.page,
+    allocatedList: state.allocated.allocatedList,
+    keyworkerList: state.allocated.keyworkerList,
+    allocatedKeyworkers: state.allocated.allocatedKeyworkers,
+    fromDate: state.allocated.fromDate,
+    toDate: state.allocated.toDate,
+    error: state.app.error
+  };
+};
+
+const mapDispatchToProps = dispatch => {
+  return {
+    unallocatedListDispatch: (list) => {
+      dispatch(setUnallocatedList(list));
+    },
+    allocatedDetailsDispatch: (allocatedList, keyworkerList) => {
+      dispatch(setAllocatedDetails(allocatedList, keyworkerList));
+    },
+    manualOverrideDispatch: (allocatedKeyworkers) => {
+      dispatch(manualOverride(allocatedKeyworkers));
+    },
+    manualOverrideDateFilterDispatch: (dateName, date) => {
+      dispatch(manualOverrideDateFilter(dateName, date));
+    },
+    setCurrentPageDispatch: (page) => {
+      dispatch(setCurrentPage(page));
+    },
+    setErrorDispatch: (error) => {
+      dispatch(setError(error));
+    }
+  };
 };
 
-export default AllocateParent;
+const AllocateParentContainer = connect(mapStateToProps, mapDispatchToProps)(AllocateParent);
```

Comment:
AllocateParentContainer is our 1 container component. Have left the non-container AllocateParent available for simple component testing